src/serialization/tokenizer.hpp

```diff
@@ -173,5 +186,5 @@ class tokenizer
 	token previous_token_;
 #endif
 	buffered_istream in_;
-	char char_types_[128];
+	token_category char_types_[START_EXTENDED_ASCII];
```

Comment:
std::array?

Reply:
It gives an error about an incomplete type.

Reply:
#include <array>?